tests/authcrypt.log

```diff
@@ -0,0 +1,4 @@
+plaintext message: [0-9a-fA-F]*
+ciphertext: [0-9a-fA-F]*
+decrypted: [0-9a-fA-F]*
+DONE


```

Comment:
These are because the template is made out of a serial output. It is not hand made. We can remove ```\r``` for readability but that will require manual effort of making the logs readable.